app/scripts/angular-apimock.js

```diff
@@ -68,7 +69,28 @@ angular.module('apiMock', [])
 			return keys;
 		}
 
-		// Taken from Angular 1.4.x: https://github.com/angular/angular.js/blob/f13852c179ffd9ec18b7a94df27dec39eb5f19fc/src/Angular.js#L296
+		// TODO: IE8: remove when we drop IE8/Angular 1.2 support.
```

Comment:
The hint is right there in the comment: we support Angular 1.2. So then we need to support whatever browsers it supports, which includes IE8.